apps/cli-daemon/src/app/workspace/workspace.controller.ts

```diff
@@ -3,33 +3,45 @@ import {
   TargetDefinition,
 } from '@angular-devkit/core/src/workspace';
 import {
+  Body,
   Controller,
   Get,
   InternalServerErrorException,
   Logger,
   NotFoundException,
   Param,
+  Post,
 } from '@nestjs/common';
 
-import { WorkspaceSettingsService } from './workspace-settings.service';
+import { SessionService } from '../session/session.service';
+
+import { WorkspaceConnectDto } from './dto';
+import { WorkspaceService } from './workspace.service';
 
 @Controller('workspace')
 export class WorkspaceController {
   private readonly logger = new Logger(WorkspaceController.name);
 
   constructor(
-    private readonly workspaceSettingsService: WorkspaceSettingsService
+    private readonly sessionService: SessionService,
+    private readonly workspaceService: WorkspaceService
   ) {}
 
+  @Post('connect')
+  async connect(@Body() body: WorkspaceConnectDto): Promise<void> {
```

Comment:
you can just destruct the "path" from the parameter

Reply:
I think this endpoint should also be called 'workspace-path' but as @Post

Reply:
The name 'workspace-path' has no sense in this case.

Reply:
why not? aren't you "settings the workspace path"? same as you "getting the workspace path"?

Reply:
actually no, 'm not just setting some path somewhere, 'm connecting the workspace to the system by setting CWD.